plugins/woocommerce/src/Blocks/BlockPatterns.php

```diff
@@ -252,6 +252,10 @@ public function register_ptk_patterns() {
 	 * @return array The parsed patterns.
 	 */
 	private function parse_categories( array $patterns ) {
+		if ( ! isset( $patterns['categories'] ) || ! is_array( $patterns['categories'] ) ) {
```

Comment:
Should we get some tests in place for this?  It looks like we could test through `register_ptk_patterns` which doesn't yet have any tests.

Reply:
This is an hotfix. I will work on the payload validation before set the transient.

Reply:
Yes @joshuatf we definitely should imo. I was thinking we address this as a follow-up given the time sensitive nature of it. Unless you think otherwise?

Reply:
That's fine.  Just want to make sure we have an issue to track these things so we don't kick any more technical debt down the road.